okapi-common/src/main/java/org/folio/okapi/common/ModuleId.java

```diff
@@ -140,11 +141,11 @@ public String toString() {
   }
 
   /**
-   * static comparsion
+   * Static module comparison.
```

Comment:
"static" is in the method signature and should not been repeated in the javadoc summary fragment.
It does not compare modules, it compares modul ids.
Can you rename the parameters from i1 to moduleId1 and i2 to moduleId2?
```suggestion
   * Module id comparison.
```